resources/views/welcome.blade.php

```diff
@@ -909,149 +909,36 @@
-                            <div
-                                class="flex size-12 shrink-0 items-center justify-center rounded-full bg-[#FF2D20]/10 sm:size-16">
-                                <svg class="size-5 sm:size-6" viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg">
-                                    <path fill="#FF2D20"
-                                        d="M369.842 28.045l-56.086 69.44 33.695 21.06 55.35-68.527-32.957-21.973zM194.64 44.265h.002L172.63 77.28l240.73 150.455 22.01-33.015-73.81-46.132zM407 92.843v-20.52l-46 53.653 12.887 8.283zm-238 3.394v26.774l67.117 41.948 14.37-17.792L169 96.238zm96.863 60.54L45.2 429.983l32.958 21.973L299.56 177.838l-33.697-21.06zm56.29 35.18v26.774L407 271.763v-26.774l-84.848-53.03zM304 204.614L88.848 470.38v18.462L304 224v-19.387zm135 17.112l-14 21v15.55l14-21zM41 448.816V475.1l30 19.375v-25.66z" />
-                                </svg>
-                            </div>
-
-                            <div class="pt-3 sm:pt-5">
-                                <h2 class="text-xl font-semibold text-black dark:text-white">Touch: Sacral Exhibtion
-                                </h2>
-
-                                <p class="mt-4 text-sm/relaxed">
-                                    The Slovak National Gallery's digital collection of artworks, including paintings,
-                                    sculptures, and other artifacts, available for public viewing and research.
-                                </p>
-                            </div>
-
-                            <svg class="size-6 shrink-0 self-center stroke-[#FF2D20]" xmlns="http://www.w3.org/2000/svg"
-                                fill="none" viewBox="0 0 24 24" stroke-width="1.5">
-                                <path stroke-linecap="round" stroke-linejoin="round"
-                                    d="M4.5 12h15m0 0l-6.75-6.75M19.5 12l-6.75 6.75" />
-                            </svg>
-                        </a>
-
-                        <a href="https://stex-touch.webumenia.sk"
-                            class="flex items-start gap-4 rounded-lg bg-white p-6 shadow-[0px_14px_34px_0px_rgba(0,0,0,0.08)] ring-1 ring-white/[0.05] transition duration-300 hover:text-black/70 hover:ring-black/20 focus:outline-none focus-visible:ring-[#FF2D20] lg:pb-10 dark:bg-zinc-900 dark:ring-zinc-800 dark:hover:text-white/70 dark:hover:ring-zinc-700 dark:focus-visible:ring-[#FF2D20]">
-                            <div
-                                class="flex size-12 shrink-0 items-center justify-center rounded-full bg-[#FF2D20]/10 sm:size-16">
-                                <svg class="size-5 shrink-0 self-center stroke-[#FF2D20]" xmlns="http://www.w3.org/2000/svg" fill="none"
-                                    viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="size-6">
+                        @foreach ($projects as $project)
+                            <a href="{{$project['slug']}}"
```

Comment:
```suggestion
                            <a href="{{$project->slug}}"
```

default approach to [access model attributes](https://laravel.com/docs/11.x/eloquent#retrieving-models)